src/client.ts

```diff
@@ -151,7 +151,7 @@ class CookiecordClient extends Client {
                     throw new Error(`Module ${file} doesn't have a default export`);
                 }
             } catch (error) {
-                const constructor: new() => unknown = error!.constructor;
+                const constructor = (error as Function).constructor;
```

Comment:
This is worse because `as` force-casts so less type-checking is performed.

Reply:
![image](https://user-images.githubusercontent.com/43092101/141688378-94a58fec-bf80-47be-a009-dcdc1fc78173.png)

you can cast to any, let me know, I will make the changes, because error is unknown.